### PR TITLE
Azure guide to resizing disks

### PR DESCRIPTION
**Pull request by DavidWoffendin**

### CODE REVIEW

File: articles/azure/azs-how-resize-disks.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+toc_mdlink: azs-how-resize-disks.md
+---
+
+# How to resize virtual machine disks
+
+In this article we will cover the process of resizing disks in the Azure portal. This will be needed if the VM requires more storage space as you can increase current disk capacity rather than adding more disks.
+
+## Steps to changing disk size via portal
+
+1. Power down or detach disk from VM.
```
Comment (suehighmoor) [inline articles/azure/azs-how-resize-disks.md]: Power down or detach the disk from the VM.

File: articles/azure/azs-how-resize-disks.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  ![Detach disks](images/azs-browser-detach-disk.png)
+2. Navigate to the resource group.
+3. Select disk.
```
Comment (suehighmoor) [inline articles/azure/azs-how-resize-disks.md]: Select the disk that you want to resize.

File: articles/azure/azs-how-resize-disks.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+4. Alter the disk size (Note the max size for a disk is 1023gb for a single disk).
```
Comment (suehighmoor) [inline articles/azure/azs-how-resize-disks.md]: In the **Size** field, enter the new size for the disk.

Comment (suehighmoor) [inline articles/azure/azs-how-resize-disks.md]: I would put the bit about the max size as a separate note:

> [!NOTE]
> The maximum size for a single disk is 1023 GiB.

(Don't forget to indent the note by four spaces)

File: articles/azure/azs-how-resize-disks.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  ![Detach disks](images/azs-browser-change-disk.png)
+5. Save changes.
```
Comment (suehighmoor) [inline articles/azure/azs-how-resize-disks.md]: Save your changes.

File: articles/azure/azs-how-resize-disks.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+6. Power back on the VM or reattach the disk.
+  ![Detach disks](images/azs-browser-attach-disk.png)
+
+These steps will work for both non managed and managed VM's.
```
Comment (suehighmoor) [inline articles/azure/azs-how-resize-disks.md]: I'd move this to the top, just before the numbered listed starts.

And get rid of the apostrophe in VMs

File: articles/azure/azs-how-resize-disks.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+toc_mdlink: azs-how-resize-disks.md
+---
+
+# How to resize virtual machine disks
+
+In this article we will cover the process of resizing disks in the Azure portal. If your VM requires more storage space, you can increase the current disk capacity, rather than adding more disks. You can do this in the Azure portal.
+
+## Resizing disk size via the portal
+
+These steps will work for both non managed and managed VM's.
```
Comment (BaileyLawson) [inline articles/azure/azs-how-resize-disks.md]: Should be unmanaged instead on non managed

File: articles/azure/azs-how-resize-disks.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+    ![Changing disk size](images/azs-browser-change-disk.png)
+  
+5. Save your changes.
+
+6. Power back on the VM or reattach the disk.
+
+    ![attaching vm disks](images/azs-browser-attach-disk.png)
+
+[!NOTE]
+The maximum size for a single disk is 1023 GiB.
```
Comment (BaileyLawson) [inline articles/azure/azs-how-resize-disks.md]: Is there a source for this? As far as I know disks can go up to 4TB

File: articles/azure/azs-how-resize-disks.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+4. In the **Size** field, enter the new size for the disk.
+
+    ![Changing disk size](images/azs-browser-change-disk.png)
+  
+5. Save your changes.
+
+6. Power back on the VM or reattach the disk.
+
+    ![attaching vm disks](images/azs-browser-attach-disk.png)
```
Comment (BaileyLawson) [inline articles/azure/azs-how-resize-disks.md]: Uppercase A

File: articles/azure/azs-how-resize-disks.md (hunk 6 above)
Comment (DavidWoffendin) [reply articles/azure/azs-how-resize-disks.md]: done

File: articles/azure/azs-how-resize-disks.md (hunk 8 above)
Comment (DavidWoffendin) [reply articles/azure/azs-how-resize-disks.md]: done

File: articles/azure/azs-how-resize-disks.md (hunk 7 above)
Comment (DavidWoffendin) [reply articles/azure/azs-how-resize-disks.md]: changed to 4tb

File: articles/azure/azs-how-resize-disks.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+
+    ![Changing disk size](images/azs-browser-change-disk.png)
+  
+5. Save your changes.
+
+6. Power back on the VM or reattach the disk.
+
+    ![Attaching vm disks](images/azs-browser-attach-disk.png)
+
+[!NOTE]
```
Comment (suehighmoor) [inline articles/azure/azs-how-resize-disks.md]: Notes need to have > before each line:

> [!NOTE]
> The maximum size for a single disk is 4TB.

File: articles/azure/azs-how-resize-disks.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+toc_mdlink: azs-how-resize-disks.md
+---
+
+# How to resize virtual machine disks
+
+In this article we will cover the process of resizing disks in the Azure portal. If your VM requires more storage space, you can increase the current disk capacity, rather than adding more disks. You can do this in the Azure portal.
+
+## Resizing disk size via the portal
+
+These steps will work for both unmanaged and managed VM's.
```
Comment (suehighmoor) [inline articles/azure/azs-how-resize-disks.md]: Remove the apostrophe in VMs

File: articles/azure/azs-how-resize-disks.md [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+
+4. In the **Size** field, enter the new size for the disk.
+
+    ![Changing disk size](images/azs-browser-change-disk.png)
+  
+5. Save your changes.
+
+6. Power back on the VM or reattach the disk.
+
+    ![Attaching vm disks](images/azs-browser-attach-disk.png)
```
Comment (suehighmoor) [inline articles/azure/azs-how-resize-disks.md]: VM should be uppercase

Comment (DavidWoffendin) [reply articles/azure/azs-how-resize-disks.md]: done

File: articles/azure/azs-how-resize-disks.md (hunk 10 above)
Comment (DavidWoffendin) [reply articles/azure/azs-how-resize-disks.md]: done

File: articles/azure/azs-how-resize-disks.md (hunk 9 above)
Comment (DavidWoffendin) [reply articles/azure/azs-how-resize-disks.md]: done

File: articles/azure/azs-how-resize-disks.md [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+
+In this article we will cover the process of resizing disks in the Azure portal. If your VM requires more storage space, you can increase the current disk capacity, rather than adding more disks. You can do this in the Azure portal.
+
+## Resizing disk size via the portal
+
+These steps will work for both unmanaged and managed VMs.
+
+1. Power down or detach the disk from the VM.
+
+    ![Detaching disks from vm](images/azs-browser-detach-disk.png)
```
Comment (suehighmoor) [inline articles/azure/azs-how-resize-disks.md]: vm => VM

File: articles/azure/azs-how-resize-disks.md [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+
+    ![Changing disk size](images/azs-browser-change-disk.png)
+  
+5. Save your changes.
+
+6. Power back on the VM or reattach the disk.
+
+    ![Attaching VM disks](images/azs-browser-attach-disk.png)
+
+> [!NOTE]
```
Comment (suehighmoor) [inline articles/azure/azs-how-resize-disks.md]: Indent this line and the next line (by four spaces)

Comment (DavidWoffendin) [reply articles/azure/azs-how-resize-disks.md]: done

**Pull request closed without merging**